documentation/feature/medical-system.md

```diff
@@ -6,238 +6,329 @@ group: feature
-Surgical kit is used to stich bandaged wounds so they will never open again. Depending on module options, surgical kit can be used only in specified situations, specified place and by specified person.
-
-### 3.2 Advanced medical system - recommended gear
+
+##### 2.2.2.3 Cardiac arrest
+
+A patient will enter cardiac arrest when:</br>
+- The heart rate is below 20.
+- The heart rate is above 200.
+- The systolic blood pressure is above 260.
+- The diastolic blood pressure is below 40 and the heart rate is above 190.
+- The systolic blood pressure is above 145 and the heart rate is above 150.
+
+
+#### 2.2.3 Treating the patient
+This is a step by step guide, follow the steps from 1 to 6 in order unless stated otherwise.
+
+
+- **Step 1:** Is the patient responsive?
+ - **Yes:** Ask him if he has wounds / he is in pain and act accordingly.
+ - **No:** Go to step 2.
+
+
+- <b>Step 2:</b> Does the patient have a pulse?
+ - **Yes:** Go to step 3.
+ - **No:** If you are alone provide CPR, if you have someone else get him to do CPR while you treat the patient's wounds. skip to step 3 or 4 depending on the situation.
  
-* Soldier:
-  * 4 x Bandage (basic)
-  * 3 x Bandage (elastic)
-  * 3 x Bandage (packing)
-  * 3 x QuikClot
-  * 1 x Morphine Autoinjector
-  * 1 x Tourniquet
-
-* Combat First Responder (CFR):
-  * 10-15 x Bandage (basic)
-  * 15-20 x Bandage (elastic)
-  * 5-8 x Bandage (packing)
-  * 10-15 x QuikClot
-  * 3 x Tourniquet
-  * 4 x Saline IV (500ml)
-  * 5 x Morphine Autoinjector
-  * 5 x Epinephrine Autoinjector
-  * 8 x Atropine Autoinjector
-
-* Medic:
-  * 10-15 x Bandage (basic)
-  * 15-20 x Bandage (elastic)
-  * 8 x Bandage (packing)
-  * 10-15 x QuikClot
-  * 5 x Tourniquet
-  * 6 x Saline IV (500ml)
-  * 8 x Morphine Autoinjector
-  * 8 x Epinephrine Autoinjector
-  * 12 x Atropine Autoinjector
-  * 1-3 x *Surgical kit*
-  * 1-3 x *Personal Aid Kit*
-
-* Paramedic:
-  * 10-15 x Bandage (basic)
-  * 15-20 x Bandage (elastic)
-  * 8 x Bandage (packing)
-  * 10-15 x QuikClot
-  * 5 x Tourniquet
-  * 2 x Saline IV (500ml)
-  * 3 x Blood IV (1000ml) - Taken from vehicle inventory only when needed
-  * 3 x Plasma IV (1000ml) - Taken from vehicle inventory only when needed
-  * 8 x Morphine Autoinjector
-  * 8 x Epinephrine Autoinjector
-  * 12 x Atropine Autoinjector
-  * 1-3 x *Surgical kit*
-  * 1-3 x *Personal Aid Kit*
-
-*medical item* - number of items that should be carried depends on module settings
-
-## Dependencies
+
+- **Step 3:** Is the patient wounded?
+ - **Yes**: Treat the wounds.
+ - **No:** Skip this step.
+  
+
+- **Step 4:** Did the patient lose blood?
+ - **Yes and normal blood pressure:** Use saline / blood IVs to restore the blood of the patient.
+ - **Yes and low blood pressure:** Use plasma IVs to stabilise the blood pressure and then use saline / blood IVs.
```

Comment:
Do plasma vs  saline/blood really behave differently now? Is there a difference between saline and blood?